.github/workflows/enforce-labels.yml

```diff
@@ -0,0 +1,13 @@
+name: Enforce labels
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled, unlabeled]
+
+jobs:
+  enforce-label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: yogevbd/enforce-label-action@2.2.2
+      with:
+        BANNED_LABELS: "don't merge"
```

Comment:
It's a good practice to [include an empty line](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) at the end of all text files, but this is a very minor detail. :nerd_face: